spalp.api.php

```diff
@@ -17,6 +17,6 @@
  */
 function hook_spalp_app_ids_alter(&$ids) {
```

Comment:
if the purpose of this PR is to transform hooks to events, then this hook doenst need to be here.